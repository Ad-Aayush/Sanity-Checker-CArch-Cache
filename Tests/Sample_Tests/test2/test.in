load ./Tests/Sample_Tests/test2/input.s
break 12
break 14
show-stack
run
show-stack
step
run
step
show-stack
run
show-stack
