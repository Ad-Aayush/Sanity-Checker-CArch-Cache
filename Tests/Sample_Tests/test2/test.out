Breakpoint set at line 12
Breakpoint set at line 14
Call Stack:
main:1
Executed addi x10, x0, 2; PC=0x00000000
Executed lui sp, 0x50; PC=0x00000004
Executed jal x1, fact; PC=0x00000008
Executed addi sp, sp, -16; PC=0x00000010
Executed sd x1, 8(sp); PC=0x00000014
Executed sd x10, 0(sp); PC=0x00000018
Executed addi x5, x10, -1; PC=0x0000001c
Executed blt x0, x5, L1; PC=0x00000020
Executed addi x10, x10, -1; PC=0x0000030
Execution stopped at breakpoint
Call Stack:
main:3
fact:14
Executed jal x1, fact; PC=0x00000034
Executed addi sp, sp, -16; PC=0x00000010
Executed sd x1, 8(sp); PC=0x00000014
Executed sd x10, 0(sp); PC=0x00000018
Executed addi x5, x10, -1; PC=0x0000001c
Executed blt x0, x5, L1; PC=0x00000020
Executed addi x10, x0, 1; PC=0x00000024
Executed addi sp, sp, 16; PC=0X00000028
Execution stopped at breakpoint
Executed jalr x0, x1(0); PC=0x0000002c
Call Stack:
main:3
fact:14
fact:12
Executed addi x6, x10, 0; PC=0x00000038
Executed ld x10, 0(sp); PC=0x0000003c
Executed ld x1, 8(sp); PC=0x00000040
Executed addi sp, sp, 16; PC=0x00000044
Executed addi x20, x0, 0; PC=0x00000048
Executed addi x8, x0, 0; PC=0x0000004c
Executed add x8, x8, x6; PC=0X00000050
Executed addi x20, x20, 1; PC=0x00000054
Executed bne x20, x10, mul; PC=0x00000058
Executed add x8, x8, x6; PC=0X00000050
Executed addi x20, x20, 1; PC=0x00000054
Executed bne x20, x10, mul; PC=0x00000058
Executed add x10, x8, x0; PC=0x0000005c
Executed jalr x0, x1(0); PC=0x00000060
Executed beq x0, x0, Exit; PC=0x0000000c
Executed add x0, x0, x0; PC=0x00000064
Empty Call Stack: Execution complete
